source/go.txt

```diff
@@ -56,6 +56,29 @@ Connect to MongoDB Atlas
 See `Usage <https://github.com/mongodb/mongo-go-driver#usage>`__
 for more detail.
 
+Versioned API
+-------------
+
+You can use the `Versioned API
+<https://docs.mongodb.com/v5.0/reference/versioned-api/>`__ feature
+starting with MongoDB Server version 5.0 and Go Driver version 1.6. When
+you use the Versioned API feature, you can update your driver or server
+without worrying about backward compatibility issues for any commands
+covered by the Versioned API. To use this feature, construct a MongoDB
+client instance, specifying a version of the Versioned API:
+
+.. code-block:: go
+
+   // Replace <connection string> with your MongoDB deployment's connection string.
+   uri := "<connection string>"
+   
+   // Set the version of the Versioned API on the client.
+   serverAPIOptions := options.ServerAPI(options.ServerAPIVersion1)
+   clientOptions := options.Client().ApplyURI(uri).SetServerAPIOptions(serverAPIOptions)
+   ctx := context.TODO()
```

Comment:
Nitpick: let's move this line before the comment "// Set the version..." since context.TODO has nothing to do with that comment.

Reply:
accepted